### ISSUE TITLE
[1.16.4] Texture issues when standing in Liquified Neon

### ISSUE THREAD

**Issue by qrss** [mcenderdragon/Futurepack-API#732]
Forge version: 35.1.28
Futurepack version: 21.0.4

When standing in Liquified Neon, the hotbar texture turns white.

Also, the icon of the Paralyze status effect you get seems to have a missing texture as a background.

![2021-01-03_05 35 54](https://user-images.githubusercontent.com/32312262/103471853-dbe2e000-4d85-11eb-8856-574489dff6e7.png)

![2021-01-03_05 40 00](https://user-images.githubusercontent.com/32312262/103471892-77745080-4d86-11eb-85ab-f9971c469bff.png)

**Comment by mcenderdragon** [comment]
https://github.com/MinecraftForge/MinecraftForge/issues/7575 its a vanilla bug with custom shaders

**Comment by qrss** [comment]
Hmm, is Liquified Neon actually obtainable in survival?

I haven't seen it in dungeons and there is no crafting recipe.

If it is not obtainable in survival, then this bug doesn't really matter right now.

**Comment by Yirok42** [comment]
I agree with qrss, but the effect should also be reachable while being attacked by, I forgot what they are called, but the robot zombies from the dungeons (the ones which drop zombie cores as they die). If one is not wearing a full suit of composite armor, then the paralyze and some what blinding affect happen there too. Though I have never at that time thought to look at my inventory.

**Comment by mcenderdragon** [comment]
> I agree with qrss, but the effect should also be reachable while being attacked by, I forgot what they are called, but the robot zombies from the dungeons (the ones which drop zombie cores as they die). If one is not wearing a full suit of composite armor, then the paralyze and some what blinding affect happen there too. Though I have never at that time thought to look at my inventory.

umm, while the paralyze effekt is aktive you cant use items anymore meaning once hit you will 100% die.
